packages/node_modules/nav-frontend-popover/_popover-input.example.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import { Input } from 'NavFrontendModules/nav-frontend-skjema';
+import { guid } from 'NavFrontendModules/nav-frontend-js-utils';
+// eslint-disable-next-line
+import Popover from './';
+
+export default class PopoverInputEksempel extends Component {
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            anker: undefined
+        };
+
+        this.popoverId = guid();
+        this.inputRef = React.createRef();
+    }
+
+    togglePopover = (anker) => {
+        const newState = Object.assign({}, this.state);
+        newState.anker = (newState.anker) ? undefined : anker;
+        this.setState(newState);
```

Comment:
Kan ikke alt her egentlig erstattes med `this.setState({ anker: this.state.anker ? undefined : anker });`